python/lsst/ap/association/__init__.py

```diff
@@ -27,3 +27,4 @@
 from .packageAlerts import *
 from .diaPipe import *
 from .transformDiaSourceCatalog import *
+from .filterDiaSourceCatalog import *
```

Comment:
We haven't been good about keeping these imports alphabetical, but you could fix that on this ticket.

Reply:
I tried re-arranging the imports, but after one point, it started complaining about circular imports. If this needs to be addressed, we can probably have another ticket.